src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -245,6 +245,22 @@ public function orWhere($column, $operator = null, $value = null)
     }
 
     /**
+     * Exclude one or many columns from being in the result colelction.
+     *
+     * @param  array $columns
+     * @return $this
+     */
+    public function exclude($columns)
+    {
+        $wantedColumns = array_diff(
+                $this->query->columns,
+                (array) $columns
+            );
+
+        return $this->select($wantedColumns);
+    }
+
```

Comment:
Please add tests.

Reply:
Ok I'm working on it now.

Reply:
@tillkruss Can you guide me? I usually use PHPUnit for my laravel applications but I don't know how to do it to my forked repo.